sdks/python/apache_beam/testing/analyzers/perf_analysis.py

```diff
@@ -103,13 +105,14 @@ def run_change_point_analysis(params, test_id, big_query_metrics_fetcher):
         timestamps=timestamps,
         min_runs_between_change_points=min_runs_between_change_points)
 
+  # TODO: remove before merging.
```

Comment:
alternative: use logging.debug if you think it will be helpful in the future to keep this logging.